webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -16055,6 +16055,42 @@ var userOverride = {
                 } else if (messageType === 'APPLOZIC_19') {
                     IS_MCK_USER_DEACTIVATED = true;
                     // events.onUserDeactivated();
+                } else if (messageType === "APPLOZIC_25") {
+                    var userId = resp.message.split(",")[0];
+                    var status = resp.message.split(",")[1];
+                    var lastSeenAtTime = resp.message.split(",")[2];
+                    if(CURRENT_GROUP_DATA.conversationAssignee != userId){
+                        return;
+                    }
+                    var statusToSet = "";
+                    var isAgentOffline;
+                    switch(parseInt(status)){
```

Comment:
can we create some array or json structure in such a way where the code is as simple as the following in just 1 line without the switch:
statusToSet = USER_STATUS[status]